specs/hash.table

```diff
@@ -7,7 +7,6 @@ schema([
     Column("sha1", TEXT, "SHA1 hash of provided filesystem data"),
     Column("sha256", TEXT, "SHA256 hash of provided filesystem data"),
 ])
-attributes(utility=True)
```

Comment:
If possible, this should keep the `hash` table until we can refactor out `./core/hash.cpp`.